### ISSUE TITLE
Python  for set globals()[''] variables having undefined name warning message

### ISSUE THREAD

**Issue by avatar-lavventura** [flycheck/flycheck#1654]
I am using flycheck for for Python. For global files that I set using ( globals()['variable']), and later used, it gives the following warning message: `undefined name <variable> [F821]`

Example:

```
globals()['var1'] = "hello_world"
print(var1) # undefined name 'fID' [F821]
```

---------

It underlines and colors it with red color.

<img width="176" alt="Screen Shot 2019-11-25 at 11 19 42" src="https://user-images.githubusercontent.com/18537398/69523683-91354880-0f75-11ea-8227-d81a57ac62e8.png">

**Comment by cpitclaudel** [comment]
This is not a Flycheck issue (Flycheck is just a display layer; it gets errors from individual linters or "checkers", which are separate programs).  You can see which linter is reporting this error using `C-c ! l` to show the error list.

**Comment by avatar-lavventura** [comment]
Thanks it is coming from `python-flake8`. They don't have any issue tab on their github page.
